app/models/subject.rb

```diff
@@ -2,7 +2,7 @@
 
 class Subject < ApplicationRecord
   validates :title, presence: true, length: { minimum: 5 }
-  validates :body, presence: true
+  validates :body, presence: true, length: {minimum: 20}
```

Comment:
can you integrate https://codeclimate.com/ in your project. It will give you comments about inconsistent style instantly

Reply:
Done, integrated